src/cc_helpers.cc

```diff
@@ -353,18 +354,27 @@ bool certifier::framework::cc_trust_data::save_store() {
   int size_protected_blob= serialized_store.size() + max_pad_size_for_store;
   byte protected_blob[size_protected_blob];
 
-  byte pkb[cc_helper_symmetric_key_size];
-  if (!get_random(8 * cc_helper_symmetric_key_size, pkb))
+  byte pkb[max_symmetric_key_size_];
+  memset(pkb, 0, max_symmetric_key_size_);
+
+  int num_key_bytes = cipher_key_byte_size(symmetric_key_algorithm_.c_str());
+  if (num_key_bytes <=0) {
+    printf("save_store: can't get key size\n");
```

Comment:
Can you not use `__func__` instead of hard-coding the method's name `save_store` in the message?

I find these preprocessor macros easier to maintain, ... if ever the method's name were to change.

Same comment applies to L366 and L377 below.